samples/system-test/similarProducts.v1p3beta1.test.js

```diff
@@ -49,7 +51,7 @@ describe(`similar products`, () => {
       }" "${localPath}" "${filter[0]}"`,
       cwd
     );
-
+    console.log(output);
```

Comment:
Are we supposed to have all of the console.logs in the tests?

Reply:
Definitely not :)

Reply:
Fixed